src/westpa/core/propagators/executable.py

```diff
@@ -95,15 +95,20 @@ def restart_writer(path, segment):
     '''Prepare the necessary files from the per-iteration HDF5 file to run ``segnment``.'''
     try:
         restart = segment.data.pop('iterh5/restart', None)
-        if restart is None:
-            raise ValueError('restart data is not present')
+        # Making an exception for start states in iteration 1
+        if segment.n_iter != 1:
+            if restart is None:
+                raise ValueError('restart data is not present')
 
         d = BytesIO(restart[:-1])  # remove tail protection
         with tarfile.open(fileobj=d, mode='r:gz') as t:
             t.extractall(path=path)
-
     except Exception as e:
-        log.warning('could not write restart data for {}: {}'.format(str(segment), str(e)))
+        if segment.n_iter != 1:
+            log.warning('could not write restart data for {}: {}'.format(str(segment), str(e)))
+        else:
+            d = BytesIO()
+            log.warning('In iteration 1, so assuming you\'re trying not to read a start state from per-iteration HDF5 file.')
```

Comment:
Here, why don't we specifically catch the ValueError you raise above (or we can raise a custom exception). It's totally plausible other unrelated stuff could break in iteration 1, so let's be parsimonious and minimal about what exceptions we try to handle

Reply:
The big exception block, I can only assume (as it's written by @SHZ66) is there to catch other FileExistsError and such from the BytesIO/tarball as well. I broke the ValueError to a specific except block, hopefully making it more clear. Maybe She can comment.